termcat-ld/README.md

```diff
@@ -0,0 +1,3 @@
+TERMCAT-LD are experiments towards the conversion of the terminological glossaries in Terminologia Oberta from the TERMCAT centre into RDF.
+Contact: Patricia Martín Chozas (patricia.martin@upm.es) 
+Github user: pmchozas
```

Comment:
Next time you update this, the markdown formatting could be improved with a list, newlines between items, or trailing whitespace.